### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by kaffeeringe** [kai-jacobsen/remindr#2]
Ich stell mir das am Ende so vor, dass man beim Bearbeiten einer Seite eine "Meta-Box" hat, in der man einstellt, ob/wann man erinnert werden will.

Ich weiß gar nicht, ob man dann eine einsehbare Liste aller Reminder benötigt. Aber zur Entwicklung ist das sicher gut.

**Comment by kai-jacobsen** [comment]
Ja, das hatte ich auch im Kopf, schließt sich aber gegenseitig nicht aus.
Da fragt sich nur, ob die Box für alle Post Types auftaucht, oder ob man das konfigurieren muss...
aber richtig, sollte berücksichtigt werden

**Comment by kaffeeringe** [comment]
Eine Erinnerung kann ja nie schaden. Aber vielleicht kann man in den Einstellungen den Standard für jeden Post-Type einstellen. Dann schaltet man eben einen auf "Nie".
